lib/registry.js

```diff
@@ -127,12 +127,16 @@ class RegistryFetcher extends Fetcher {
     }
 
     const packument = await this.packument()
-    let mani = await pickManifest(packument, this.spec.fetchSpec, {
+    const mani = await pickManifest(packument, this.spec.fetchSpec, {
       ...this.opts,
       defaultTag: this.defaultTag,
       before: this.before,
     })
-    mani = rpj.normalize(mani)
+      // TODO: https://github.com/npm/package-json/pull/96
+      // new PackageJson().fromContent(pkg).normalize()
+      .then((pkg) => new PackageJson().fromJSON(JSON.stringify(pkg)).normalize())
```

Comment:
This is not a later `TODO`. This needs to be fixed before landing this PR. I have an open PR to add this to `@npmcli/package.json` https://github.com/npm/package-json/pull/96